docs/compilers/CSharp/Compiler Breaking Changes - DotNet 6.md

```diff
@@ -0,0 +1,13 @@
+﻿## This document lists known breaking changes in Roslyn in C# 9.0 introduced with .NET 6.
```

Comment:
Consider adding a link for this document in <https://github.com/dotnet/docs/blob/master/docs/csharp/whats-new/breaking-changes.md>.